lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -85,19 +90,56 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                     return YARNClientConfiguration.ConfigurationModule.Build();
                 case YARNRest:
                     return YARNClientConfiguration.ConfigurationModuleYARNRest.Build();
+                case HDInsightLocal:
+                    {
+                        // To run against HDInsight please replace placeholders below, with actual values for
+                        // connection string, container name (available at Azure portal) and HDInsight 
+                        // credentials (username and password)
+                        const string connectionString = "ConnString";
+                        const string containerName = "foo";
+                        return
+                            new MultiRuntimeConfigurationBuilder().AddRuntime(RuntimeName.Yarn)
```

Comment:
This seems like a `Builder` with everything, it won't really scale with more environments. This might be better off as a Builder that builds from more Builders.